_topic_map.yml

```diff
@@ -168,8 +168,11 @@ Topics:
   File: web-console
 - Name: Configuring the web console
   File: configuring-web-console
+- Name: Developer perspective
+  File: odc-about-developer-perspective-in-the-web-console
```

Comment:
@Preeticp  I don't think we should use 'in', 'the' as part of file names, contexts or anchors.